src/main/java/org/prebid/server/json/deserializer/CommaSeparatedStringAsListOfIntegersDeserializer.java

```diff
@@ -49,6 +49,7 @@ private static void reportWrongTokenException(JsonParser parser, Deserialization
     private static List<Integer> parseList(String value) throws NumberFormatException {
         return Arrays.stream(value.split(","))
                 .map(StringUtils::strip)
+                .filter(StringUtils::isNotBlank)
```

Comment:
Should we use StringUtils.isNumeric(), or something similar? 
(with additionaal test for this case)

Reply:
I disagree. We need to report the error to the client and reject the request.